charts/topolvm/values.yaml

```diff
@@ -402,6 +406,13 @@ controller:
   #  rollingUpdate:
   #    maxSurge: 50%
   #    maxUnavailable: 50%
+  #
+  # For non-leader-election mode, you can use this non-HA non-conflicting strategy:
+  #  type: Recreate
+  #  rollingUpdate:
+  #    maxSurge: 0
+  #    maxUnavailable: 1
+
```

Comment:
Is this part necessary?
It seems that rollingUpdate.maxSurge and rollingUpdate.maxUnavailable cannot be specified for type: Recreate